test/test_isaac_sim.py

```diff
@@ -315,3 +346,49 @@ def test_add_habitat_visual_metadata_for_articulation():
                     "habitatVisual:assetScale"
                 ).Get()
             )
+
+
+def test_bulk_scene_instance_conversion():
+    project_root_folder = repo_dir
+    scene_directory = repo_dir + "/../" + "habitat-sim/data/hssd-hab/scenes"
+    scene_output_dir = (
+        repo_dir + "/test/data/usd_conversion_data/bulk_testing_output"
+    )
+
+    if os.path.exists(scene_output_dir):
+        shutil.rmtree(scene_output_dir)
+        print(f"Directory '{scene_output_dir}' deleted successfully.")
+
+    if not os.path.exists(scene_output_dir):
+        os.mkdir(scene_output_dir)
+        gitignore_filepath = scene_output_dir + "/.gitignore"
+        with open(gitignore_filepath, "w"):
+            pass
+        print(f"File '{gitignore_filepath}' created successfully.")
+
+    scene_instance_json_list = find_scene_instance_json_files(scene_directory)
+
+    for scene_filepath in scene_instance_json_list:
+        scene_usd_filepath = make_scene_instance_usd_path(
+            scene_filepath, scene_output_dir
+        )
+
+        subprocess.run(
+            [
+                "python",
+                scene_instance_conversion_script,
+                scene_filepath,
+                project_root_folder,
+                scene_usd_filepath,
+            ]
+        )
+
+        assert True
```

Comment:
Let's do a real test here, e.g. assert that all the scenes you found got converted, i.e. test for existence of expected files.